### ISSUE TITLE
Dates Still on Blog Posts

### ISSUE THREAD

**Issue by madpenny** [eyebeam/eyebeam.org#21]
Hey Dan,

The date still shows up on blog posts. 
<img width="1190" alt="screenshot 2018-03-08 11 19 08" src="https://user-images.githubusercontent.com/31375201/37162177-98ece872-22c2-11e8-8c1d-523e6c96f0f1.png">

**Comment by dphiffer** [comment]
Yeah, I guess this seems like an essential bit of info for a blog post. But maybe the Ideas page really isn't a blog at all, in the commonly understood sense.

A different way to approach this might be to shift some things out of "Posts" and into a different custom post type, or just as a sub-Page of another section. I think this especially applies for things like the Education programs, which feel like they should live under `/education/___` rather than the top-level `/____`.

**Comment by dphiffer** [comment]
But yeah, I could also just hide the dates for everything! We have a field for editing the content in there, and perhaps in some cases it might be the date.
